prepros.config

```diff
@@ -0,0 +1,932 @@
+{
```

Comment:
What is it?

Reply:
Sorry, it is for using SCSS but it is not necessary because SCSS is not used.